Project.toml

```diff
@@ -36,7 +36,7 @@ RecipesBase = "0.7,1"
 StatsBase = "0.32,0.33"
 TextParse = "0.9.1,1"
 WeakRefStrings = "0.6"
-julia = "1"
+julia = "1.6"
```

Comment:
`ismutabletype` requires Julia 1.7

Reply:
Does it make sense to define a mutability check function that checks the Julia version and runs a compatible version of the test?

Reply:
The source of `ismutabletype` looks pretty simple.  We just need something like:

```Julia
ismutabletype = if VERSION < v"1.7"
    function ismutabletype(@nospecialize(t::Type))
        t = unwrap_unionall(t)
        return isa(t, DataType) && t.name.flags & 0x2 == 0x2
    end
else
    Base.ismutabletype
end
```